### ISSUE TITLE
Animation of thorns of Bristleback Thornweavers are broken

### ISSUE THREAD

**Issue by laytya** [Wall-core/Everlook-Bugtracker#272]
After end of effect of thorns are stay some last animation on ground.

![1](https://user-images.githubusercontent.com/20452393/195285113-3b14699c-1181-47fe-926b-bf9f4d0e2183.JPG)
![2](https://user-images.githubusercontent.com/20452393/195285127-f02a5b2c-eb04-4422-be38-8b42e37a43cc.JPG)
![3](https://user-images.githubusercontent.com/20452393/195285128-ca0f3df9-0bda-43a8-b89d-bf6231ad5432.JPG)

It gone only after exit from game.

**Comment by laytya** [comment]
Same for animation of frost nova of Bristleback Water Seeker

![4](https://user-images.githubusercontent.com/20452393/195290328-4c3d60d3-3845-4549-a7b0-b3f0650ec389.JPG)

or even from my frost nova

**Comment by Wall-core** [comment]
Client side; there may be a fix but this is low priority as it's visually handled in client.
